spec/lib/web_of_science/query_author_spec.rb

```diff
@@ -21,122 +16,78 @@
+      expect(query_blank_orcid.name_query).to be_valid
+      num_uids_total = query_blank_orcid.uids.size
+      num_uids_name = query_blank_orcid.name_query.uids.size
+      expect(num_uids_total).to eq num_uids_name # the only uids are from the name query
+      expect(query_blank_orcid.uids.size).to be > 450
     end
   end
 
-  describe '#empty_fields' do
-    it 'has collections with empty fields' do
-      expect(query_author.send(:empty_fields)).to include(collectionName: String, fieldName: [''])
+  describe '#uids with only the orcid query valid' do
+    subject(:query_blank_author) { described_class.new(author_blank_name) }
+
+    # The VCR fixture is ~ 150 records at the time it was recorded;
+    # if the VCR cassette is updated, this value could change
+    # and this spec assumes it's only going to get larger
+    it 'returns uids from the orcid query only' do
+      expect(query_blank_author).to be_valid
+      expect(query_blank_author.orcid_query).to be_valid
+      expect(query_blank_author.name_query).not_to be_valid
+      num_uids_total = query_blank_author.uids.size
+      num_uids_orcid = query_blank_author.orcid_query.uids.size
+      expect(num_uids_total).to eq num_uids_orcid  # the only uids are from the orcid query
+      expect(num_uids_total).to be > 140
     end
   end
 
-  describe '#quote_wrap' do
-    it 'wraps strings in double quotes' do
-      expect(query_author.send(:quote_wrap, %w[a bc def])).to eq %w["a" "bc" "def"]
-    end
-
-    it 'deduplicates and removes empties' do
-      expect(query_author.send(:quote_wrap, %w[a bc a bc].concat(['']))).to eq %w["a" "bc"]
-    end
-
-    it 'removes quotes in a name' do
-      expect(query_author.send(:quote_wrap,
-                               ['peter', 'peter paul',
-                                'peter "paul" mary'])).to eq ['"peter"', '"peter paul"', '"peter paul mary"']
-    end
-  end
-
-  context 'with a user with valid first names' do
-    it 'indicates it is a valid query' do
-      expect(query_author).to be_valid
-    end
-  end
-
-  context 'with a user with no valid first names' do
-    it 'indicates that name with a period for a first name is not a valid query' do
-      expect(query_period_author).not_to be_valid
-    end
-
-    it 'indicates that a name with a space for a first name is not a valid query' do
-      expect(query_space_author).not_to be_valid
-    end
-
-    it 'indicates that a name with a blank for a first name is not a valid query' do
-      expect(query_blank_author).not_to be_valid
-    end
-  end
-
-  context 'for a single alternate identity with invalid data' do
-    describe '#names' do
-      let(:author_one_identity) { create :author }
-      let(:bad_alternate_identity) { create :author_identity }
-
-      before do
-        bad_alternate_identity.update(first_name: '.')
-        author_one_identity.author_identities << bad_alternate_identity
-      end
+  describe '#uids with both the orcid and name queries invalid' do
```

Comment:
all of these tests were just moved to the new name specific class below where the methods were moved to, they were not actually deleted: `QueryName`